toxcore/logger.c

```diff
@@ -89,7 +89,7 @@ Logger *logger_new (const char *file_name, LOG_LEVEL level, const char *id)
 {
 #ifndef LOGGING /* Disabled */
     return NULL;
-#endif
+#else
```

Comment:
This is actually a bad idea, as it causes the logging code to be compiled only when LOGGING is enabled, which is probably rare, risking the code breaking due to rare exercise.